marahgh/assignment3/wordSearch.cpp

```diff
@@ -0,0 +1,212 @@
+#include <iostream>
+#include <vector>
+#include <algorithm>
+using namespace std;
+
+/*A helper function:returns true if the given word
+was found in the given vector, and false otherwise*/
+bool isFoundInVector(vector<string> words, string word) {
+  if (std::find(words.begin(), words.end(), word) != words.end()) {
+    return true;
+  }
+  return false;
+}
+
+/*The grid class,
+contains lower-case letters from whichever alphabet the dictionary uses.*/
+class Grid {
+  char** matrix;
+  int rows;
+  int cols;
+  bool empty;

```

Comment:
For things that are now immediately clear (like `empty`), you can add a comment describing what this member variable is for.